receiver/mysqlreceiver/metadata.yaml

```diff
@@ -57,6 +57,19 @@ attributes:
     value: kind
     description: The thread count type.
     enum: [cached, connected, created, running]
+  schema:
+    value: schema
+    description: The schema of the object.
+  io_waits_operations:
+    value: operation
+    description: The io_waits operation type.
+    enum: [delete, fetch, insert, update]
+  io_waits_name:
+    value: name
+    description: Object name for event or process.
```

Comment:
Is this always a table name?

Reply:
That's hard to deduce. The description here is directly copied from [Telegraf docs](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/mysql#tags).
According to [this](https://dev.mysql.com/doc/mysql-perfschema-excerpt/5.6/en/performance-schema-table-io-waits-summary-by-table-table.html) (`OBJECT_NAME` column, which is described [here](https://dev.mysql.com/doc/mysql-perfschema-excerpt/5.6/en/performance-schema-events-waits-current-table.html)), it is true for "table I/O objects". I guess that indices and table are such objects in every case?

Reply:
I decided to change "object" to "table" here.